samples/py_custom_module/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+if(NOT IREE_BUILD_PYTHON_BINDINGS)
+  return()
+endif()
+
+iree_py_test(
+  NAME
+    decode_secret_message_sample
+  SRCS
+    "decode_secret_message.py"
```

Comment:
Thanks for the test! I just verified that this is running on the new GitHub Actions CI:

https://github.com/iree-org/iree/runs/7604942229?check_suite_focus=true
```
992/1014 Test #1156: iree/samples/py_custom_module/decode_secret_message_sample ...............................................................   Passed    0.62 sec
```